addons/mail/views/mail_activity_views.xml

```diff
@@ -190,6 +204,18 @@
         </field>
     </record>
 
+    <record id="mail_activity_action_view_tree" model="ir.actions.act_window.view">
+        <field name="sequence" eval="1"/>
+        <field name="view_mode">tree</field>
+        <field name="act_window_id" ref="mail.mail_activity_action"/>
```

Comment:
@tde-banana-odoo @seb-odoo 
Hello, You did not set any view_id for this record. Was it on purpose ? 
I have a strange bug in inherited views on a model that use the activity mixin. This bug seems fixed once I set the tree view.

Reply:
Hello @Whenrow, I had to double check when merging but apparently this is on purpose, `view_id` is not necessary when using `ir.actions.act_window.view` because it comes in replacement.

See note for the `view_ids` param a bit below on the doc: https://www.odoo.com/documentation/master/reference/actions.html#window-actions-ir-actions-act-window.

Honestly I don't know much about this stuff so if you find another way to make it work here and that also works for your case feel free to change.